pkg/metrics/parser.go

```diff
@@ -94,6 +94,11 @@ func (p *Parser) parseMetric(metricPath string, deviceID string, value interface
 	} else {
 		return Metric{}, fmt.Errorf("got data with unexpectd type: %T ('%s')", value, value)
 	}
+
+	if cfg.MQTTValueScale != 0 {
+		metricValue = metricValue / cfg.MQTTValueScale
```

Comment:
From a UX perspective I'm a bit confused that the `MQTTValueScale` is used as divisor would have expected that the `MQTTValueScale` is used as factor in a multiplication.

Reply:
it's more about this proportion i put in the readme

`The scale of the metric in a MQTT JSON message (mqtt_value : scale = prom_value : 1)`

so considering `prom_value` as the "output" then `prom_value = mqtt_value / scale`

but yeah it would be the same thing

Reply:
Yeah, I would consider `prom_value = mqtt_value * scale` if we keep the name scale.